intercom_2n/utils.py

```diff
@@ -0,0 +1,33 @@
+import os
+import socket
+import struct
+import fcntl
+
+def get_lan_ip():
+    try:
+        ip = socket.gethostbyname(socket.gethostname())
+        if ip.startswith("127.") and os.name != "nt":
+            interfaces = ["eth0", "eth1", "eth2", "wlan0", "wlan1", "wifi0", "ath0", "ath1", "ppp0"]
```

Comment:
What if users will not have such interfaces and use different names or having multiple LAN interfaces. Could also be a configuration option for the plugin, right?